qcsrc/defs.qc

```diff
@@ -394,36 +426,80 @@ float	UPDATE_TEMP		= 3;
 
 // entity effects
 
+#ifdef __QW__ // Engines support different entity effects
+float	EF_BRIGHTLIGHT 	= 4;
+float	EF_DIMLIGHT 	= 8;
+float   EF_FLAG1		= 16;
+float   EF_FLAG2		= 32;
+float 	EF_BLUE			= 64;	// Blue Globe effect for Quad
+float	EF_RED			= 128;
+#else
 float	EF_BRIGHTFIELD	= 1;
 float	EF_MUZZLEFLASH 	= 2;
 float	EF_BRIGHTLIGHT 	= 4;
 float	EF_DIMLIGHT 	= 8;
-
+#endif
 
 // messages
 float	MSG_BROADCAST	= 0;		// unreliable to all
 float	MSG_ONE			= 1;		// reliable to one (msg_entity)
 float	MSG_ALL			= 2;		// reliable to all
 float	MSG_INIT		= 3;		// write to the init string
 
+#ifdef __QW__ // Multicast is QW only
+float   MSG_MULTICAST   = 4;		// for multicast() call
+#endif
+
+
+// message levels
+float   PRINT_LOW               = 0;            // pickup messages
```

Comment:
Nitpick: #define these maybe

Reply:
This is code directly copied from the QuakeWorld codebase. Should we refactor all of defs.qc to use #define statements instead of global constants?

Reply:
Personally I would, just for cleanliness and prog size. Consider it low priority though, I'm fine if you'd want to proceed without it